### ISSUE TITLE
Strip output file timings

### ISSUE THREAD

**Issue by berquist** [cclib/cclib#241]
There is currently a discussion online about the usage of output files in supplementary information for publications, and I think it's relevant for us.

https://twitter.com/janhjensen/status/697370524838715392

> is it allowed to supply Gaussian input/output files as supplementary material? #compchem

https://twitter.com/janhjensen/status/699861956937777152

> Gaussian: yes to "include relevant parts of output" Do not include timings cc @olexandr @ghutchis @adrian_roitberg @Dr_GHill @theochemtheo

https://twitter.com/ghutchis/status/699921078852825089

> @janhjensen @ChristofJaeger @olexandr @adrian_roitberg @Dr_GHill if that's really the worry, strip the timing from the output by default

https://twitter.com/janhjensen/status/699921853574463488

> @ghutchis @ChristofJaeger @olexandr @adrian_roitberg @Dr_GHill yep. Lines of concern contain "cpu", easy to delete https://t.co/ozM3CsyG0a

It's probably too much work and maybe not a good idea to permanently alter history, but it should be done for current and future files, either by hand or using a commit hook.

**Comment by langner** [comment]
I don't think we really need to alter history, be we could remove all timings in one swoop and then all releases after that commit won't be problematic. I would not remove actually, but replace with a placeholder along the lines of "This line contained timing information and has been removed".

We can also do this for regression files.

**Comment by berquist** [comment]
I like that. There is a chance that the parser relies on those lines being present for something like `skip_line` (or worse, introduce more bugs), so preserving the number of lines is a good idea.

**Comment by baoilleach** [comment]
Here's my 2c. Strip 'em only when making the release. I don't really like
the idea of changing files our users have presented.

**Comment by langner** [comment]
> Here's my 2c. Strip 'em only when making the release. I don't really like
> the idea of changing files our users have presented.

I agree, and preferably when switching to version 2.x (when that happens).
